app/src/main/java/com/byeboo/app/domain/repository/auth/TokenRepository.kt

```diff
@@ -1,5 +1,14 @@
 package com.byeboo.app.domain.repository.auth
 
+import com.byeboo.app.core.model.auth.TokenEntity
+import kotlinx.coroutines.flow.Flow
+
 interface TokenRepository {
-    suspend fun getUserId(): Long?
+    suspend fun getAccessToken(): Flow<String>
```

Comment:
`Flow` 자체가 콜드 스트림으로 알고 있는데 `suspend`가 굳이 필요 있을까요? 
```suggestion
    fun getAccessToken(): Flow<String>
```
밑에도 동일하고요! 아니면 suspend fun + `Flow`를 쓰신 특별한 이유가 있는지 궁금합니다!

Reply:
원래 String으로 받았었는데 리이슈 api 오류 때문에 Flow로 바꾸고 나서 `suspend` 를 못 지웠습니다... `suspend` 삭제할게요 감사합니다!